### PR TITLE
bump bitcoinjs-lib dependency

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: src/request.js [hunk 1]
```diff
@@ -12,36 +12,33 @@ const bitcoinjs_lib_1 = require('bitcoinjs-lib');
...
-})();
+  static codeToMessage(code) {
+    return (
+      this.messageMap[code] ||
+      'Something went wrong when requesting the payjoin endpoint.'
+    );
+  }
+}
 exports.PayjoinEndpointError = PayjoinEndpointError;
+PayjoinEndpointError.messageMap = {
```
Comment (Overtorment) [inline src/request.js:28]: I had to run build script and commit the result, otherwise tests complained

**Comment by junderw** [comment]
Remove package-lock.json please. Thanks.

**Comment by junderw** [comment]
Also could you add package-lock.json to .gitignore?

**Comment by junderw** [comment]
wait... you can commit files in gitignore???

It's already in there...

**Comment by junderw** [comment]
nvm I was able to delete it

**Comment by Overtorment** [comment]
why not include package-lock file in repo?

btw, to commit gitignored files you need to add it with `-f` flag

**Comment by junderw** [comment]
> why not include package-lock file in repo?

https://dev.to/gajus/stop-using-package-lock-json-or-yarn-lock-3ddi

A member of BitcoinJS went through all the libraries adding package-locks all around and the amount of headaches it caused are countless... on the other hand, the number of times not having a lockfile made extra work or had some bad effects was.... maybe one time?

Lockfiles are not needed for libraries.

Applications (Like Blue Wallet) are what should use lockfiles.

**Pull request merged**